.github/actions/build-image/action.yaml

```
@@ -0,0 +1,31 @@
+name: Build container image
+description: Build container image and check vulnerabilities
+
+inputs:
+  image_tag:
+    description: The tag of the image
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+    - name: Build container image
+      uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825
+      with:
+        tags: ${{ env.image_tag }}
```

```suggestion
        tags: ${{ inputs.image_tag }}
```
I guess this should change as well